src/commands/lambda/flare.ts

```
@@ -1,14 +1,239 @@
+
+        return 1
+      }
+    }
+
+    // Get Lambda function configuration
+    this.context.stdout.write('\n🔍 Getting Lambda function configuration...\n')
+    const lambdaClientConfig: LambdaClientConfig = {
+      region,
+      credentials: this.credentials,
+    }
+    const lambdaClient = new LambdaClient(lambdaClientConfig)
+    let config
+    try {
+      config = await getLambdaFunctionConfig(lambdaClient, this.functionName ?? '')
+    } catch (err) {
+      this.context.stderr.write(
+        commonRenderer.renderError(`Unable to get Lambda function configuration: ${err.message}`)
+      )
+
+      return 1
+    }
+
+    // Print config
+    const configStrColored = util.inspect(config, false, undefined, true)
+    this.context.stdout.write(`\n${configStrColored}\n`)
+    if (this.isDryRun) {
+      this.context.stdout.write('\n🚫 The configuration was not sent as it was executed in dry run mode.\n')
+
+      return 0
+    }
+
+    // Send data to Datadog
+    const folderPath = path.join(process.cwd(), FLARE_OUTPUT_DIRECTORY)
+    const filePath = path.join(folderPath, FUNCTION_CONFIG_FILE_NAME)
+    const zipPath = path.join(folderPath, ZIP_FILE_NAME)
+    const configStrUncolored = JSON.stringify(config, undefined, 2)
+    try {
+      await this.writeFile(folderPath, filePath, configStrUncolored)
+      await this.zipContents(filePath, zipPath)
+      this.context.stdout.write('\n🚀 Sending to Datadog Support...\n')
+      await this.sendToDatadog(zipPath)
+      this.deleteFolder(folderPath)
+    } catch (err) {
+      this.context.stderr.write(err.message)
+
+      return 1
+    }
+
+    return 0
+  }
+
+  /**
+   * Write the function config to a file
+   * @param folderPath
+   * @param filePath
+   * @param data
+   * @throws Error if the file cannot be written
+   */
+  private writeFile = async (folderPath: string, filePath: string, data: string) => {
+    if (!fs.existsSync(folderPath)) {
+      try {
+        fs.mkdirSync(folderPath)
```

This should just throw the error as `throw Error('message here')`